v2/client.go

```diff
@@ -1122,3 +1130,164 @@ func (c *Client) NewSubAccountTransferHistoryService() *SubAccountTransferHistor
 func (c *Client) NewListUserUniversalTransferService() *ListUserUniversalTransferService {
 	return &ListUserUniversalTransferService{c: c}
 }
+
+// Create virtual sub-account
+func (c *Client) NewCrtVirtSubAccService() *CrtVirtSubAccService {
```

Comment:
To keep consistent code style, it is recommended to use full words instead of abbreviations.

```suggestion
func (c *Client) NewCreateSubAccountService() *CreateSubAccountService {
```

Reply:
ok

Reply:
ok